src/gtk/toga_gtk/widgets/combobox.py

```diff
@@ -0,0 +1,50 @@
+from gi.repository import Gtk
+from travertino.size import at_least
+
+from .base import Widget
+
+
+class ComboBox(Widget):
+
+    def create(self):
+        self.native = Gtk.ComboBoxText.new_with_entry()
+        self.native.interface = self.interface
+        self.native.connect('changed', self._on_change)
+        self.native.connect('show', lambda event: self.rehint())
+
+    def _on_change(self, widget):
```

Comment:
We're moving (slowly) towards identifying platform-native handlers with a prefix that identifies them - in this case, `gtk_on_change`.